### DISCUSSION TITLE
iOS: search favorites by repo owner

### DISCUSSION THREAD

**Discussion by mcritz** [community/community#3611] [Mobile]
I'd like to add favorites by repo owner. 

Ex: 

A search for "vapor" should show repos that belong to "vapor" like "https://github.com/vapor/vapor"

**Comment by PranamBhat** [comment]
Hi @mcritz 

You can always filter the best match like this:

![image](https://user-images.githubusercontent.com/32120033/112586158-43f31080-8e21-11eb-956d-840b872f20ca.png)

**Reply by mcritz** [reply]
Please note that I am using “iOS:” to indicate I am using the native iOS GitHub client. And I’m using the Favorites feature of that app which has no equivalent on the GitHub.com web site.
